Database/Database.csproj

```diff
@@ -30,10 +30,6 @@
 	<None Update="Default database initialisation.sql">
 		<CopyToOutputDirectory>Always</CopyToOutputDirectory>
 	</None>
-    <None Update="HoMM5 ToE creature costs v1.0.db">
```

Comment:
Why did you remove the database from the project file?

Reply:
Good catch! 

Definitely not intentional. Bringing it back.